internal/docs/mdx.go

```diff
@@ -0,0 +1,49 @@
+package docs
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/spf13/cobra"
+)
+
+func GenMdxFile(cmd *cobra.Command, dir string) error {
```

Comment:
Could we keep a consistent naming between the `yaml` and `mdx` version?
```suggestion
func GenMdxTree(cmd *cobra.Command, dir string) error {
```